GTG/gtk/data/task_editor.ui

```diff
@@ -19,7 +19,7 @@
             <property name="receives_default">False</property>
             <property name="tooltip_text" translatable="yes">Mark this task as not to be done anymore</property>
             <property name="action_name">app.dismiss</property>
-            <property name="text" translatable="yes">Dismiss Task</property>
+            <property name="text" translatable="yes">Dismiss</property>
```

Comment:
I don't think we should make this shorter for translating purposes. Only if it makes sense for the dialog conceptually.

Reply:
On one hand I'm like "this could have been left alone, yes", on the other hand I was also envisioning that this item present in the task editor's "..." menubutton could/would eventually become a headerbar button on the same footing as the "Mark as Done" button, in  which case the " Task" word precision is redundant too...